src/transformers/cache_utils.py

```diff
@@ -817,17 +822,21 @@ def __init__(self, config: PretrainedConfig, max_batch_size: int, max_cache_len:
         )
 
         self.key_cache: List[torch.Tensor] = []
-        self.value_cache: List[torch.Tensor] = []
+        self.val_cache: List[torch.Tensor] = []
```

Comment:
nitpicking: `value_cache` is a naming convention we use throughout our code and our docs, let's stick to the original name :)

Reply:
Love consistence! Addressed